src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -283,6 +285,14 @@
               <!-- NB: no documents so no divider needed -->
             </template>
 
+            <!-- is this a staff only filing? -->
+            <template v-else-if="isStaffFiling(filing.filingType)" id="staff-filings-info">
+              <p class="info-text">{{filing.notationOrOrder}}</p>
+              <p class="info-text my-0">Court Order Number: {{filing.fileNumber}}</p>
+              <p class="info-text my-0">{{filing.planOfArrangement}}</p>
```

Comment:
Worth creating a sub-component?

Reply:
errr i don't think so. Tbh there is no visual design for this either, just a UX pin. 
If there is some standard format for this in the future perhaps. Right now it's as simple as this 🤷